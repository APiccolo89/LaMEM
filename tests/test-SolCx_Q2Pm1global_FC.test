EXEC            = ../bin/LaMEM -Benchmark_SolCx -mumax 1 -nnode_x 9 -nnode_y 3 -nnode_z 9 -vpt_element Q2Pm1_global -DisplayAverageQuantities -StokesSolver 3 -vs_ksp_max_it 104
NPROCS          = 1
TEST_TYPE       = tolerance_diff_pl,1.0e-4
