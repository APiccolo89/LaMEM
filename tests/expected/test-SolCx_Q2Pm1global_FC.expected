# -------------------------------------------------------------------------- 
#                   Lithosphere and Mantle Evolution Model                   
#              Current Revision: 4812 - 2013-10-01 23:57:18		     
#  Modified items: LaMEM_Particles.c, Elements.c, Assembly_FDSTAG.c, fdstag/JacRes.c, 
#     Compiled: Date: Oct  9 2013 - Time: 23:02:54 - Mode:  	    
# -------------------------------------------------------------------------- 
 Stokes solver             : Fully Coupled
 Velocity solver           : Direct (Build-in direct solver)
 Input is assumed to be in non-dimensional units 
 Element-type used         : Q2Pm1_global;  [Q1P0 Q2Pm1_global Q2Pm1_local Q1Q1 FDSTAG] (change with -vpt_element)
 Total # of cpu's          : 1 
 Total # of velocity dof's : 729 
 Computing mean material per element using arithmetic averaging from integration points 
 Employing code in an Eulerian manner 
 Gravity angle with z-axis =  0 
 Approximate Aspect ratio of cells [dx/dz, dy/dz] = [1 4] 
 WARNING: At least 3 particles required in x-direction for Q2 elements; I have increased this 
 WARNING: At least 3 particles required in y-direction for Q2 elements; I have increased this 
 WARNING: At least 3 particles required in z-direction for Q2 elements; I have increased this 
 Number of tracers/cell in [x,y,z]-direction = [3,3,3] 
 NumSurfaceNodes=1  nz=9 
 # Not employing nonlinear iterations 
 BC employed: BC.[Left Right; Front Back; Lower Upper]=[1 1; 1 1; 1 1] 
# Initial time step: 0.001 years 
 Input is assumed to be in non-dimensional units 
 Fine Grid parameters, nel=[4,1,4], cpu=[1,1,1] 
# Finished initializing Stokes matrices : 0.014008 s 
# Not adding random noise to initial particle distribution. 
Initialized particles 
#rank=0, Attributing 432 particles to elements took 0.00974798 s 
The total # of particles present is 432 
 *** Using a falling block setup *** 
 Falling block coordinates:  [left,right]=[0.250000,0.750000]; [front,back]=[0.000000,0.000000]; [bottom,top]=[0.250000,0.750000] 
Saved initial particle distribution to directory ./InitialParticles 
The total memory used by integration points is 1.140480e-01 MB
The max memory used by integration points is 1.140480e-01 MB
The min memory used by integration points is 1.140480e-01 MB
#  Interpolating meshes and temperatures took 0.00014782 s 


 --------------------------------------------------------------------------------  
 
             PERFORMING BENCHMARK OF LaMEM VERSUS ANALYTICAL SOLUTION              
  using:                             SolCx                                        
 
   with:      rho   = -1*sin(1*PI*z)*cos(1*PI*x)                                  
              eta   = 1 if x<x_c & eta=eta_b for x>x_c                            
    and:      x_c   = 0.5     (change with -SolCx_xc ... )                         
              eta_b = 1     (change with -mumax ...    )                          

 --------------------------------------------------------------------------------  
 
 
#  Forming FEM stiffness matrix mx,my,mz=(9, 3, 9) ...   finished. [0.016722 s]
 minDiagonalRatio=1 [Critical = 0.55; change with -CriticalDiagonalRatio]  
#  Forming stiffness matrixes took 0.016947 s 
FC: Viscosity structure info: min(0)=1.000000e+00 : max(0)=1.000000e+00 : contrast=1.000000e+00 
Using unpreconditioned (true) Stokes residual to define the FC stopping condition 
StokesResidual: Using L_inf 
Stopping conditions: 
Velocity: RTOL: 1.000000e-06	ATOL: 1.000000e-16	minATOL: 1.000000e-05		fc_ksp_min_it 4
Pressure: RTOL: 1.000000e-06	ATOL: 1.000000e-16	minATOL: 1.000000e-05		fc_ksp_min_it 4
StokesResidual: Monitor activated 
# FC: using nonzero initial guess 
  Residual norms for fc_ solve.
  # KSP Iterations      /  L2_norm             scaled_L2_norm      Linf_norm           /  ... for each   2 blocks  [fc_] 
  0 KSP Residual norms  /  4.717938938705e-02  1.747384792113e-03  1.532647963990e-02  /  0.000000000000e+00  0.000000000000e+00  0.000000000000e+00  /  [fc_] 
  1 KSP Residual norms  /  3.515616976276e-03  1.302080361584e-04  1.142067176125e-03  /  1.238971749216e-02  1.548714686520e-03  3.732550964290e-03  /  [fc_] 
  2 KSP Residual norms  /  1.269297103380e-05  4.701100382888e-07  4.123380243942e-06  /  7.737477052417e-04  9.671846315521e-05  4.620065037736e-04  /  [fc_] 
  3 KSP Residual norms  /  3.103065238248e-06  1.149283421573e-07  1.008047514247e-06  /  3.826108913155e-04  4.782636141443e-05  1.247698832650e-04  /  [fc_] 
  4 KSP Residual norms  /  2.286488487353e-06  8.468475879087e-08  7.427781433119e-07  /  3.284355380323e-04  4.105444225404e-05  1.526363166115e-04  /  [fc_] 
  5 KSP Residual norms  /  1.554781794713e-06  5.758451091530e-08  5.050792694428e-07  /  2.708343374769e-04  3.385429218462e-05  1.107876996913e-04  /  [fc_] 
  6 KSP Residual norms  /  1.187996314245e-06  4.399986349055e-08  3.859270237405e-07  /  2.367435941335e-04  2.959294926668e-05  9.508516383536e-05  /  [fc_] 
  7 KSP Residual norms  /  7.185004179934e-07  2.661112659235e-08  2.334087443488e-07  /  1.841138092343e-04  2.301422615429e-05  4.683636761561e-05  /  [fc_] 
  8 KSP Residual norms  /  7.131390474234e-07  2.641255731198e-08  2.316670740390e-07  /  1.834256149957e-04  2.292820187446e-05  5.007825958396e-05  /  [fc_] 
  9 KSP Residual norms  /  3.290956347082e-07  1.218872721142e-08  1.069085011796e-07  /  1.246050672722e-04  1.557563340902e-05  3.623966026992e-05  /  [fc_] 
 10 KSP Residual norms  /  3.290070235360e-07  1.218544531615e-08  1.068797159020e-07  /  1.245882908913e-04  1.557353636142e-05  3.595285116673e-05  /  [fc_] 
 11 KSP Residual norms  /  1.830161996648e-07  6.778377765361e-09  5.945382794683e-08  /  9.292232805706e-05  1.161529100713e-05  2.785022565616e-05  /  [fc_] 
 12 KSP Residual norms  /  1.382775871945e-07  5.121392118316e-09  4.492042363834e-08  /  8.077024828018e-05  1.009628103502e-05  2.680164047536e-05  /  [fc_] 
 13 KSP Residual norms  /  8.323270899930e-08  3.082692925900e-09  2.703920584806e-08  /  6.266467470393e-05  7.833084337991e-06  1.923133628375e-05  /  [fc_] 
 14 KSP Residual norms  /  7.629884894699e-08  2.825883294333e-09  2.478756607072e-08  /  5.999772629193e-05  7.499715786491e-06  1.933577305072e-05  /  [fc_] 
 15 KSP Residual norms  /  6.673926797977e-08  2.471824739991e-09  2.168404916319e-08  /  5.611339146943e-05  7.014173933679e-06  1.670536254476e-05  /  [fc_] 
 16 KSP Residual norms  /  5.965851823062e-08  2.209574749282e-09  1.947678894854e-08  /  5.305246357756e-05  6.631557947195e-06  1.774946498663e-05  /  [fc_] 
 17 KSP Residual norms  /  2.593166204238e-08  9.604319274957e-10  8.630362671229e-09  /  3.496422854381e-05  4.370528567976e-06  1.266661690780e-05  /  [fc_] 
 18 KSP Residual norms  /  2.704910747485e-08  1.001818795365e-09  8.471190970779e-09  /  3.272534656323e-05  4.090668320404e-06  1.268683536629e-05  /  [fc_] 
 19 KSP Residual norms  /  9.874887785133e-07  3.657365846346e-08  3.571970569946e-07  /  2.254873733723e-05  2.818592167153e-06  1.002628899031e-05  /  [fc_] 
 20 KSP Residual norms  /  4.438476049777e-07  1.643880018436e-08  1.605204965731e-07  /  2.145542548188e-05  2.681928185235e-06  9.565731766522e-06  /  [fc_] 
 21 KSP Residual norms  /  3.594301463115e-06  1.331222764117e-07  1.300213959371e-06  /  2.021377491383e-05  2.526721864229e-06  9.443117277593e-06  /  [fc_] 
 22 KSP Residual norms  /  1.339293301498e-06  4.960345561102e-08  4.844724235141e-07  /  1.972020881855e-05  2.465026102319e-06  9.203302776512e-06  /  [fc_] 
 23 KSP Residual norms  /  5.596809408408e-07  2.072892373485e-08  2.024402273199e-07  /  1.952790763594e-05  2.440988454492e-06  8.821678038269e-06  /  [fc_] 
 24 KSP Residual norms  /  5.240484386448e-07  1.940920143129e-08  1.895562926930e-07  /  1.769516298956e-05  2.211895373694e-06  7.631649129200e-06  /  [fc_] 
 25 KSP Residual norms  /  8.901642903895e-08  3.296904779220e-09  3.213763647617e-08  /  1.623908328093e-05  2.029885410116e-06  6.115123837335e-06  /  [fc_] 
 26 KSP Residual norms  /  4.465612625294e-08  1.653930601961e-09  1.614624740249e-08  /  8.104004453016e-06  1.013000556627e-06  2.594288683561e-06  /  [fc_] 
 27 KSP Residual norms  /  4.198373611537e-09  1.554953189458e-10  1.448188338068e-09  /  7.724647570809e-06  9.655809463511e-07  2.141642850803e-06  /  [fc_] 
 28 KSP Residual norms  /  3.240137431738e-08  1.200050900644e-09  1.171539808675e-08  /  6.877901308802e-06  8.597376636002e-07  2.118271375092e-06  /  [fc_] 
 29 KSP Residual norms  /  7.690359099879e-09  2.848281148103e-10  2.759213733556e-09  /  6.803419036281e-06  8.504273795352e-07  2.114366664001e-06  /  [fc_] 
 30 KSP Residual norms  /  1.117240801177e-09  4.137928893249e-11  3.721935923967e-10  /  4.532628473028e-06  5.665785591285e-07  1.406976687307e-06  /  [fc_] 
 31 KSP Residual norms  /  1.103287453468e-09  4.086249827660e-11  3.675452241987e-10  /  4.532322951324e-06  5.665403689155e-07  1.409141297745e-06  /  [fc_] 
 32 KSP Residual norms  /  3.266766211533e-09  1.209913411679e-10  1.088278776260e-09  /  4.366614883418e-06  5.458268604272e-07  1.298206449005e-06  /  [fc_] 
 33 KSP Residual norms  /  2.929190480607e-09  1.084885363188e-10  9.758200083061e-10  /  4.363140962073e-06  5.453926202592e-07  1.329203705885e-06  /  [fc_] 
 34 KSP Residual norms  /  2.101664326150e-08  7.783941948704e-10  7.001409139075e-09  /  4.222717877631e-06  5.278397347038e-07  1.116898572158e-06  /  [fc_] 
 35 KSP Residual norms  /  1.391139192591e-08  5.152367379966e-10  4.634391199152e-09  /  4.052980064802e-06  5.066225081003e-07  1.303430981330e-06  /  [fc_] 
 36 KSP Residual norms  /  3.038638685906e-07  1.125421735521e-08  1.012281190735e-07  /  3.851964890340e-06  4.814956112925e-07  1.383773814997e-06  /  [fc_] 
 37 KSP Residual norms  /  5.199447009202e-07  1.925721114519e-08  1.732125123678e-07  /  3.023632353092e-06  3.779540441365e-07  1.283668278974e-06  /  [fc_] 
 38 KSP Residual norms  /  6.553442891465e-08  2.427201070913e-09  2.183190454446e-08  /  3.002502468089e-06  3.753128085111e-07  1.230772177076e-06  /  [fc_] 
 39 KSP Residual norms  /  1.082187706063e-07  4.008102615048e-09  3.605161300592e-08  /  2.987912948062e-06  3.734891185077e-07  1.241398167454e-06  /  [fc_] 
 40 KSP Residual norms  /  2.966627519980e-08  1.098750933326e-09  9.882916492605e-09  /  2.812754707635e-06  3.515943384543e-07  1.200643207700e-06  /  [fc_] 
 41 KSP Residual norms  /  3.744884472796e-08  1.386994249184e-09  1.247557378528e-08  /  2.808082426558e-06  3.510103033197e-07  1.197321419141e-06  /  [fc_] 
 42 KSP Residual norms  /  1.162208262867e-08  4.304475047654e-10  3.871738911949e-09  /  2.198460671586e-06  2.748075839483e-07  9.007967161442e-07  /  [fc_] 
 43 KSP Residual norms  /  4.155800175925e-08  1.539185250343e-09  1.384448361411e-08  /  2.083389450554e-06  2.604236813192e-07  8.960718858389e-07  /  [fc_] 
 44 KSP Residual norms  /  3.334674802129e-10  1.235064741529e-11  1.110901699247e-10  /  1.474427581523e-06  1.843034476904e-07  6.815772673833e-07  /  [fc_] 
 45 KSP Residual norms  /  1.322847194297e-09  4.899434052952e-11  4.406885783888e-10  /  1.437443512135e-06  1.796804390169e-07  6.993303856021e-07  /  [fc_] 
 46 KSP Residual norms  /  1.689220981224e-09  6.256374004535e-11  5.627409357179e-10  /  1.272621429670e-06  1.590776787087e-07  6.416228473361e-07  /  [fc_] 
 47 KSP Residual norms  /  7.784630862598e-10  2.883196615777e-11  2.593342344067e-10  /  1.241055761434e-06  1.551319701793e-07  6.250601995434e-07  /  [fc_] 
 48 KSP Residual norms  /  1.528268695517e-09  5.660254427840e-11  5.091204714674e-10  /  1.128227799946e-06  1.410284749932e-07  5.164753580128e-07  /  [fc_] 
 49 KSP Residual norms  /  2.435266355936e-09  9.019505021985e-11  8.112794991326e-10  /  1.102042062093e-06  1.377552577616e-07  5.021677144464e-07  /  [fc_] 
 50 KSP Residual norms  /  1.887590213773e-09  6.991074865824e-11  6.287952841023e-10  /  7.784909297037e-07  9.731136621297e-08  3.591824792910e-07  /  [fc_] 
 51 KSP Residual norms  /  5.334659338223e-09  1.975799754897e-10  1.777202773816e-09  /  7.565816047910e-07  9.457270059888e-08  3.512169626026e-07  /  [fc_] 
 52 KSP Residual norms  /  5.553745969738e-10  2.056942951755e-11  1.851743330628e-10  /  6.592434389249e-07  8.240542986561e-08  3.115320217902e-07  /  [fc_] 
 53 KSP Residual norms  /  6.278941658325e-10  2.325533947528e-11  2.091388627323e-10  /  6.561532408144e-07  8.201915510180e-08  3.106982550025e-07  /  [fc_] 
 54 KSP Residual norms  /  7.316922207288e-10  2.709971187884e-11  2.389328667426e-10  /  5.006106122013e-07  6.257632652517e-08  2.222348526100e-07  /  [fc_] 
 55 KSP Residual norms  /  1.674193560583e-09  6.200716891049e-11  5.539796016008e-10  /  4.903393952123e-07  6.129242440153e-08  2.198759546769e-07  /  [fc_] 
 56 KSP Residual norms  /  9.901655970121e-11  3.667279988934e-12  2.816444226716e-11  /  4.021876768571e-07  5.027345960714e-08  1.934276830171e-07  /  [fc_] 
 57 KSP Residual norms  /  2.485571407412e-10  9.205820027451e-12  1.158760524564e-10  /  3.993686275691e-07  4.992107844614e-08  1.926001982374e-07  /  [fc_] 
 58 KSP Residual norms  /  5.343329108059e-09  1.979010780763e-10  1.798303622958e-09  /  2.802694939350e-07  3.503368674188e-08  1.247357334277e-07  /  [fc_] 
 59 KSP Residual norms  /  3.463876740637e-08  1.282917311347e-09  1.129437231171e-08  /  2.518550652364e-07  3.148188315454e-08  1.097009138195e-07  /  [fc_] 
 60 KSP Residual norms  /  3.735305332509e-08  1.383446419448e-09  1.274886003980e-08  /  2.193819936938e-07  2.742274921172e-08  9.331580448481e-08  /  [fc_] 
 61 KSP Residual norms  /  3.615199744080e-08  1.338962868178e-09  1.233893121188e-08  /  2.194343543963e-07  2.742929429953e-08  9.388007910878e-08  /  [fc_] 
 62 KSP Residual norms  /  8.047992060863e-09  2.980737800319e-10  2.746836342726e-09  /  2.157459612822e-07  2.696824516028e-08  9.546246893987e-08  /  [fc_] 
 63 KSP Residual norms  /  5.450736721002e-09  2.018791378149e-10  1.860374812330e-09  /  2.126238816649e-07  2.657798520811e-08  1.002666163625e-07  /  [fc_] 
 64 KSP Residual norms  /  5.961679149613e-09  2.208029314671e-10  2.034763059909e-09  /  2.114739086202e-07  2.643423857752e-08  1.007454400365e-07  /  [fc_] 
 65 KSP Residual norms  /  9.562392025851e-09  3.541626676241e-10  3.263711711124e-09  /  2.058971746487e-07  2.573714683109e-08  1.021617009676e-07  /  [fc_] 
 66 KSP Residual norms  /  2.277231080833e-10  8.434189188272e-12  7.772349980498e-11  /  1.763491472882e-07  2.204364341103e-08  9.861292508567e-08  /  [fc_] 
 67 KSP Residual norms  /  5.090884445598e-11  1.885512757629e-12  1.737554697062e-11  /  1.668803248050e-07  2.086004060062e-08  9.998092933158e-08  /  [fc_] 
 68 KSP Residual norms  /  3.776192330111e-10  1.398589751893e-11  1.288841024193e-10  /  1.407893566948e-07  1.759866958685e-08  7.961701600397e-08  /  [fc_] 
 69 KSP Residual norms  /  4.210382681649e-10  1.559400993203e-11  1.437033247536e-10  /  1.407472493711e-07  1.759340617139e-08  8.004226217684e-08  /  [fc_] 
 70 KSP Residual norms  /  1.075461106186e-09  3.983189282170e-11  3.670624458633e-10  /  1.275274312490e-07  1.594092890613e-08  7.904732259176e-08  /  [fc_] 
 71 KSP Residual norms  /  1.073592029059e-09  3.976266774294e-11  3.664245166962e-10  /  1.275182900978e-07  1.593978626222e-08  7.948189997614e-08  /  [fc_] 
 72 KSP Residual norms  /  7.008921857067e-10  2.595896984099e-11  2.392194385351e-10  /  1.172278981997e-07  1.465348727496e-08  7.237741647712e-08  /  [fc_] 
 73 KSP Residual norms  /  8.132691800595e-10  3.012108074295e-11  2.775744963715e-10  /  1.169421210353e-07  1.461776512941e-08  7.400018924349e-08  /  [fc_] 
 74 KSP Residual norms  /  4.499093991493e-10  1.666331107960e-11  1.535572119382e-10  /  9.383421249472e-08  1.172927656184e-08  4.933363359326e-08  /  [fc_] 
 75 KSP Residual norms  /  5.000389963090e-10  1.851996282626e-11  1.706668426117e-10  /  9.021934051497e-08  1.127741756437e-08  5.259967392250e-08  /  [fc_] 
 76 KSP Residual norms  /  4.541447894363e-10  1.682017738653e-11  1.550030701828e-10  /  8.225287348698e-08  1.028160918587e-08  4.933101634433e-08  /  [fc_] 
 77 KSP Residual norms  /  4.674024940009e-10  1.731120348152e-11  1.595278151686e-10  /  8.196067929979e-08  1.024508491247e-08  4.946854635534e-08  /  [fc_] 
 78 KSP Residual norms  /  2.703819834241e-10  1.001414753423e-11  9.227745743509e-11  /  6.476373561963e-08  8.095466952453e-09  3.597091433676e-08  /  [fc_] 
 79 KSP Residual norms  /  4.150675274386e-10  1.537287138662e-11  1.416653350904e-10  /  6.262615884814e-08  7.828269856018e-09  3.460702901103e-08  /  [fc_] 
 80 KSP Residual norms  /  2.629165423936e-10  9.737649718282e-12  8.978713801867e-11  /  4.737576692642e-08  5.921970865802e-09  2.560956732218e-08  /  [fc_] 
 81 KSP Residual norms  /  1.540027708997e-10  5.703806329617e-12  5.258038192486e-11  /  4.690062253994e-08  5.862577817493e-09  2.580197582412e-08  /  [fc_] 
 82 KSP Residual norms  /  9.623574778289e-11  3.564286954922e-12  4.175645453990e-11  /  4.379223820111e-08  5.474029775138e-09  2.233412216576e-08  /  [fc_] 
 83 KSP Residual norms  /  1.579707202639e-10  5.850767417180e-12  5.387994885217e-11  /  4.331578035430e-08  5.414472544287e-09  2.301845999581e-08  /  [fc_] 
 84 KSP Residual norms  /  1.245223946395e-09  4.611940542205e-11  5.075901861899e-10  /  2.682120881627e-08  3.352651102034e-09  1.064073931980e-08  /  [fc_] 
 85 KSP Residual norms  /  4.539520513988e-10  1.681303894070e-11  2.118419874317e-10  /  2.500330736452e-08  3.125413420565e-09  1.026477009730e-08  /  [fc_] 
 86 KSP Residual norms  /  4.892829114037e-09  1.812158931125e-10  2.453910258176e-09  /  2.351766916223e-08  2.939708645279e-09  9.519302489326e-09  /  [fc_] 
 87 KSP Residual norms  /  1.007216609092e-09  3.730431885526e-11  5.052506598925e-10  /  2.308270989266e-08  2.885338736583e-09  9.414640362024e-09  /  [fc_] 
 88 KSP Residual norms  /  8.528160209826e-10  3.158577855491e-11  4.313969306958e-10  /  2.272218597770e-08  2.840273247212e-09  8.824822108135e-09  /  [fc_] 
 89 KSP Residual norms  /  5.063892712964e-11  1.875515819616e-12  2.744200183520e-11  /  1.942998049269e-08  2.428747561587e-09  7.186128385921e-09  /  [fc_] 
 90 KSP Residual norms  /  2.169012061730e-10  8.033378006406e-12  1.162223640636e-10  /  1.806496955496e-08  2.258121194370e-09  6.569447304261e-09  /  [fc_] 
 91 KSP Residual norms  /  1.623824095972e-10  6.014163318415e-12  8.700950934631e-11  /  1.765970726186e-08  2.207463407733e-09  7.771253840102e-09  /  [fc_] 
 92 KSP Residual norms  /  1.148073819969e-10  4.252125259146e-12  6.151734046608e-11  /  1.754955613411e-08  2.193694516763e-09  7.641235466561e-09  /  [fc_] 
 93 KSP Residual norms  /  1.915530408889e-10  7.094557069958e-12  1.026400343663e-10  /  1.689785278671e-08  2.112231598339e-09  8.446703474527e-09  /  [fc_] 
 94 KSP Residual norms  /  1.402753230118e-09  5.195382333771e-11  7.516384969857e-10  /  1.669074311926e-08  2.086342889907e-09  8.287643818258e-09  /  [fc_] 
 95 KSP Residual norms  /  2.253699363416e-09  8.347034679318e-11  1.207601711980e-09  /  1.630732915306e-08  2.038416144132e-09  8.382796796432e-09  /  [fc_] 
 96 KSP Residual norms  /  3.096307491316e-10  1.146780552339e-11  1.659097166208e-10  /  1.594206813421e-08  1.992758516776e-09  7.931802490737e-09  /  [fc_] 
 97 KSP Residual norms  /  4.200981968934e-10  1.555919247753e-11  2.251015863088e-10  /  1.579138585450e-08  1.973923231812e-09  8.412228443981e-09  /  [fc_] 
 98 KSP Residual norms  /  4.389357768341e-11  1.625688062348e-12  2.351953433345e-11  /  1.074401081710e-08  1.343001352138e-09  4.449113798834e-09  /  [fc_] 
 99 KSP Residual norms  /  1.086583825003e-10  4.024384537048e-12  5.822251665855e-11  /  9.365367310279e-09  1.170670913785e-09  4.634421654940e-09  /  [fc_] 
100 KSP Residual norms  /  5.092002422882e-12  1.885926823290e-13  2.728452135354e-12  /  7.833350142786e-09  9.791687678482e-10  3.771564184145e-09  /  [fc_] 
101 KSP Residual norms  /  6.816294438370e-12  2.524553495693e-13  3.652381043667e-12  /  7.779048267512e-09  9.723810334389e-10  3.953449061635e-09  /  [fc_] 
102 KSP Residual norms  /  2.247959364801e-11  8.325775425189e-13  1.204526044452e-11  /  6.875888649379e-09  8.594860811724e-10  3.854310598445e-09  /  [fc_] 
103 KSP Residual norms  /  2.151596673524e-11  7.968876568607e-13  1.152891938987e-11  /  6.875169122168e-09  8.593961402710e-10  3.836154972852e-09  /  [fc_] 
104 KSP Residual norms  /  1.588011784354e-11  5.881525127235e-13  8.509057415525e-12  /  6.241875610401e-09  7.802344513002e-10  3.600486152146e-09  /  [fc_] 
Velocity convergence reason: RTOL 
Pressure convergence reason: RTOL 
Linear solve converged due to CONVERGED_HAPPY_BREAKDOWN iterations 104
#  Finished Stokes solve after 0.0559981 s 
------------------------------------------
Stokes solver summary: 
  divergence: 
    Div_min = 8.754998859765e-12 
    Div_max = 3.600486152236e-09 
    |Div|_2 = 6.241875605139e-09 
    |Div|_1 = 3.318237200895e-08 
  momentum: 
    mRes_min = 0.000000000000e+00 
    mRes_max = 8.508925768841e-12 
    |mRes|_2 = 1.587998336043e-11 
    |mRes|_1 = 1.629425382611e-10 
  velocity: 
    u_min   = -2.557215500548e-02 
    u_max   = 2.557215614478e-02 
    |u|_2   = 2.775977344444e-01 
    |u|_1   = 4.606927386858e+00 
  pressure: 
    p_min   = -4.271699132365e-01 
    p_max   = 4.271524941689e-01 
    |p|_2   = 1.577864151965e+00 
    |p|_1   = 9.259202914642e+00 
------------------------------------------
The total memory used by integration points is 1.140480e-01 MB
The max memory used by integration points is 1.140480e-01 MB
The min memory used by integration points is 1.140480e-01 MB
#  Solving for velocity took 0.074333 s 
The total memory used by integration points is 1.140480e-01 MB
The max memory used by integration points is 1.140480e-01 MB
The min memory used by integration points is 1.140480e-01 MB


 --------------------------------------------------------------------------------  

  LaMEM STOKES BENCHMARK RESULTS:                                                 

       dx = 0.125        [node spacing in x-direction   ]
       dy = 0.5        [node spacing  in y-direction   ] 
       dz = 0.125        [node spacing  in z-direction   ] 

  |Vx |_1 = 7.61413e-05 
  |Vy |_1 = 1.96694e-09 
  |Vz |_1 = 7.42113e-05 

  |u  |_1 = 7.91991e-05   [velocity                      ]
  |P  |_1 = 0.00364334   [pressure                      ]
  |e  |_1 = 0.00184268   [strain rate                   ]
  |P  |_2 = 9.25319e-10   [pressure                      ]

 --------------------------------------------------------------------------------  

The total memory used by integration points is 1.140480e-01 MB
The max memory used by integration points is 1.140480e-01 MB
The min memory used by integration points is 1.140480e-01 MB
#  Forming temperature stiffness matrix mx,my,mz=(9, 3, 9) ...  finished. 
#  Forming temperature stiffness matrixes took 0.00236011 s 
#  Solving for temperature took 0.270307 s 
The total memory used by integration points is 1.140480e-01 MB
The max memory used by integration points is 1.140480e-01 MB
The min memory used by integration points is 1.140480e-01 MB
The total memory used by particles is 1.209600e-01 MB
The max memory used by particles is 1.209600e-01 MB
The min memory used by particles is 1.209600e-01 MB
#  Updating properties @ integration points and particles took 0.00236702 s 
AdvectionMethod = 0 
  Starting particle advection 
  Finished particle advection 
# Volume-averaged quantities: 
# Txx = -2.54988e-19 Exx = -1.27494e-19 Volume = 3.375 Mu_xx=1 
# Tyy = -3.92528e-26 Eyy = -1.96264e-26 Volume = 3.375 Mu_yy=1 
# Tzz = -9.63735e-20 Ezz = -4.81868e-20 Volume = 3.375 Mu_zz=1 
# Txz = 2.50363e-06 Exz = 2.50363e-06 Volume = 3.375 Mu_xz=0.5 
# Txy = 5.34721e-11 Exy = 5.34721e-11 Volume = 3.375 Mu_xy=0.5 
# Tyz = -4.21329e-10 Eyz = -4.21329e-10 Volume = 3.375 Mu_yz=0.5 
Minimum z-level phase 1 2 
# Writing output to existing directory Timestep_000000 
# Saved MATLAB output in file Timestep_000000/AnalyticalBenchmark_SolCx.0.1000000.out 
# Saved Paraview VTS Velocity output in file AnalyticalBenchmark_SolCx_velocity_step000000-mesh.pvts 
# Saved Paraview VTS Temperature output in file AnalyticalBenchmark_SolCx_temperature_step000000-mesh.pvts 
# Saved Paraview VTS Surface Topography output in file AnalyticalBenchmark_SolCx_Surface_topography_step000000-mesh.pvts 
# Saved Paraview VTS Bottom Topography output in file AnalyticalBenchmark_SolCx_Bottom_topography_step000000-mesh.pvts 
# Saved Paraview VTS point-wise quadrature output in file AnalyticalBenchmark_SolCx_quadrature_step000000-mesh.pvts 
 Starting GetParticleNodes 
#rank=0, Attributing 432 particles to elements took 0.0111768 s 
The total # of particles present is 432 
 Finished GetParticleNodes 
 Maximum velocity  0.0255722    
 Time = 0.001, dt=0.21725   
# Finished timestep 0 out of 1 in 0.373259 s
  
# Total time required: 0.39954 s 
#PETSc Option Table entries:
-AddRandomNoiseParticles 0
-Benchmark_SolCx
-DisplayAverageQuantities
-StokesSolver 3
-fc_ksp_converged_reason
-fc_ksp_max_it 1000
-fc_ksp_min_it 4
-fc_ksp_p_atol 1.0e-16
-fc_ksp_p_min_atol 1.0e-5
-fc_ksp_p_rtol 1.0e-6
-fc_ksp_type gcr
-fc_ksp_v_atol 1.0e-16
-fc_ksp_v_min_atol 1.0e-5
-fc_ksp_v_rtol 1.0e-6
-fc_pc_fieldsplit_schur_factorization_type UPPER
-fc_pc_fieldsplit_type SCHUR
-fc_pc_type fieldsplit
-mumax 1
-nnode_x 9
-nnode_y 3
-nnode_z 9
-options_left
-ps_ksp_type preonly
-ps_pc_type jacobi
-use_stokes_monitor
-use_stokes_norm Linf
-use_stokes_residual
-vpt_element Q2Pm1_global
-vs_ksp_max_it 104
-vs_ksp_type preonly
-vs_pc_type lu
#End of PETSc Option Table entries
There are no unused options.
